.github/workflows/pythonapp.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10.0-rc.2"]
```

Comment:
This is clearly working through GHA, but on my own setup with an updated `conda`, I don't have access to 3.10 yet.

Reply:
Oh right, this package doesn't use `conda` for set up. Does it make sense to wait for 3.10 proper?